docs/a320-simvar-list.md

```diff
@@ -0,0 +1,692 @@
+# List all LVARs for the A32NX
```

Comment:
**issue**: Duplicate information, which won't be maintained by developers.
**suggestion**: Remove this file and only keep `docs/a320-simvars-table.md`

Reply:
Ok, will do. Can be recreated any time anyway.